web/src/admin.js

```diff
@@ -224,6 +224,9 @@ export function build_page() {
         automatically_unmute_topics_in_muted_streams_policy_values:
             settings_config.automatically_follow_or_unmute_topics_policy_values,
         realm_enable_guest_user_indicator: realm.realm_enable_guest_user_indicator,
+        active_user_list_dropdown_widget_name: settings_users.active_user_list_dropdown_widget_name,
+        deactivated_user_list_dropdown_widget_name:
+            settings_users.deactivated_user_list_dropdown_widget_name,
```

Comment:
Just a small note that in https://github.com/zulip/zulip/issues/26949, we'll probably end up needing only one of these, but I like the naming scheme.